### PR TITLE
【fluid api clear】Move batch norm1

### PR DESCRIPTION
**Pull request by xiaoguoguo626807**
### PR types
Others 

### PR changes
 APIs 

### Describe

move 'fluid.layers.nn.batch_norm' to 'paddle.static.nn.batch_norm'

### CODE REVIEW

File: python/paddle/static/nn/common.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```
Comment (JiabinYang) [inline python/paddle/static/nn/common.py]: why blank new line

**Review by JiabinYang — DISMISSED** [review]
LGTM

File: python/paddle/static/nn/common.py [hunk 2]
```diff
@@ -2088,6 +2089,328 @@ def deform_conv2d(
...
+    data_layout='NCHW',
+    in_place=False,
+    name=None,
+    moving_mean_name=None,
+    moving_variance_name=None,
+    do_model_average_for_mean_and_var=True,
+    use_global_stats=False,
+):
+    r"""
+    :api_attr: Static Graph
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py]: 删掉

File: python/paddle/static/nn/common.py [hunk 3]
```diff
@@ -2088,6 +2089,328 @@ def deform_conv2d(
...
+
+    2. NCHW `[batch, in_channels, in_height, in_width]`
+
+    Refer to `Batch Normalization: Accelerating Deep Network Training by Reducing
+    Internal Covariate Shift <https://arxiv.org/pdf/1502.03167.pdf>`_
+    for more details.
+
+    :math:`input` is the input features over a mini-batch.
+
+    ..  math::
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py:2196]: 在注释之前加了 `r` ，公式里就不需要使用两个斜杠了，这里公式比较多，建议构建一个官网预览确认一下

**Review by XiaoguangHu01 — DISMISSED** [review]
LGTM

File: python/paddle/static/nn/common.py [hunk 4]
```diff
@@ -2088,6 +2089,328 @@ def deform_conv2d(
...
+            If it is set to None, batch_norm will save global variance with a random name, otherwise, batch_norm
+            will save global variance with the string.
+        do_model_average_for_mean_and_var(bool, Default True): Whether parameter mean and variance should do model
+            average when model average is enabled.
+        use_global_stats(bool, Default False): Whether to use global mean and
+            variance. In inference or test mode, set use_global_stats to true
+            or is_test to true, and the behavior is equivalent.
+            In train mode, when setting use_global_stats True, the global mean
+            and variance are also used during train period.
+    Returns:
```
Comment (Ligoml) [inline python/paddle/static/nn/common.py:2274]: returns 和前面的参数部分增加一个空行吧

Comment (xiaoguoguo626807) [reply python/paddle/static/nn/common.py:2274]: done

File: python/paddle/static/nn/common.py (hunk 3 above)
Comment (xiaoguoguo626807) [reply python/paddle/static/nn/common.py:2196]: done

File: python/paddle/static/nn/common.py (hunk 2 above)
Comment (xiaoguoguo626807) [reply python/paddle/static/nn/common.py]: done

File: python/paddle/static/nn/common.py (hunk 1 above)
Comment (xiaoguoguo626807) [reply python/paddle/static/nn/common.py]: deleted

**Review by JiabinYang — APPROVED** [review]
LGTM

**Review by zyfncg — APPROVED** [review]
LGTM for Yaml check

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

**Review by momozi1996 — APPROVED** [review]
LGTM
